### PR TITLE
Ziggs/Darius ability

### PR DESCRIPTION
**Pull request by areke**
Adds node_modules to the git ignore, and an ability implementation for ziggs + darius

### CODE REVIEW

File: src/game/HexEffect.ts [hunk 1]
```diff
@@ -30,13 +34,14 @@ export class HexEffect {
...
 	stunMS: number | null
+	onCollision: (unit: ChampionUnit) => void
 
 	activated = false
 
 	constructor(source: ChampionUnit, elapsedMS: DOMHighResTimeStamp, data: HexEffectData) {
 		this.instanceID = `h${instanceIndex += 1}`
-		this.startsAtMS = elapsedMS + data.spell.castTime! * 1000
-		this.activatesAfterMS = data.spell.missile!.travelTime!
+		this.startsAtMS = elapsedMS + (data.spell.castTime ?? DEFAULT_CAST_TIME) * 1000
```
Comment (areke) [inline src/game/HexEffect.ts:43]: darius doesn't have these values in the data file. i substituted with estimates

**Comment by ky-is** [comment]
Thanks very much for the contribution! Will review this week.

**Comment by ky-is** [comment]
btw it's ideal to make a single PR per champion. Feel free to branch from the first one to reuse any changes shared between both in future!

**Review by ky-is — CHANGES_REQUESTED** [review]

**Comment by ky-is** [comment]
Thanks @areke, I updated from main and merged. I'll make a few followup commits but looks great!

**Comment by mark-klein-addepar** [comment]
thank you!! sorry, busy with work lately, but hoping to help out a bunch more this week.

**Comment by areke** [comment]
thank you!! meant to follow up, but been busy with work lately. hoping to help out some more this week :)

**Comment by ky-is** [comment]
No worries, just didn't want this to fall too far behind. Sounds great!

**Pull request merged**